feature-libs/product-configurator/common/components/configurator-cart-entry-bundle-info/configurator-cart-entry-bundle-info.component.spec.ts

```diff
@@ -453,52 +453,7 @@ describe('ConfiguratorCartEntryBundleInfoComponent', () => {
         fixture.detectChanges();
       });
 
-      it('should display in desktop mode', () => {
-        spyOn(breakpointService, 'isUp').and.returnValue(of(true));
-        fixture.detectChanges();
-        CommonConfiguratorTestUtilsService.expectElementPresent(
-          expect,
-          htmlElem,
-          '.cx-item-infos.open'
-        );
-
-        CommonConfiguratorTestUtilsService.expectNumberOfElements(
-          expect,
-          htmlElem,
-          '.cx-item-info',
-          1
-        );
-
-        CommonConfiguratorTestUtilsService.expectNumberOfElements(
-          expect,
-          htmlElem,
-          '.cx-item-price span.cx-item',
-          1
-        );
-
-        CommonConfiguratorTestUtilsService.expectElementToContainText(
-          expect,
-          htmlElem,
-          '.cx-item-price span.cx-item',
-          '$1,000.00'
-        );
-
-        CommonConfiguratorTestUtilsService.expectNumberOfElements(
-          expect,
-          htmlElem,
-          '.cx-item-quantity span.cx-item',
-          1
-        );
-
-        CommonConfiguratorTestUtilsService.expectElementToContainText(
-          expect,
-          htmlElem,
-          '.cx-item-quantity span.cx-item',
-          '5'
-        );
-      });
-
-      it('should display in mobile mode', () => {
```

Comment:
I left only unit tests for mobile (aka. "compact") view, and changed test description to just "should display"